samples/ClassroomLabs/Modules/Library/Tests/Utils.ps1

```diff
@@ -0,0 +1,63 @@
+Import-Module $PSScriptRoot\..\Az.LabServices.psm1
+
+$rgName = 'TestPerm'
+$rgLocation = 'West Europe'
+$labName = 'FastLab'
+$laName = 'LATest'
+$imgName = 'CentOS-Based*'
+$maxUsers = 2
+$usageQuota = 30
+$usageAMode = 'Restricted'
+$shPsswd = $false
+$size = 'Medium'
+$title = 'Advancing Differentiation Workshop'
+$descr = 'Bringing it to the 21st Century'
+$userName = 'test0000'
+$password = 'Test000'
```

Comment:
I wonder if we should have another way to handle credentials, like storing in keyvault and reading them?  Or generating and saving them somewhere?  Can do that in the future...

Reply:
Yep, it does look bad but rather harmless as written. Agree to revisit in the future.